### PR TITLE
Merge release v0.4.3

### PR DESCRIPTION
**Pull request by sea212**
### What does it do?
Merges main into a clone of [branch release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/release-v0.4.3) as a preparation to merge [branch release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/release-v0.4.3) into main.

### CODE REVIEW

File: runtime/battery-station/src/lib.rs [hunk 1]
```diff
@@ -102,7 +102,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 51,
+    spec_version: 52,
     impl_version: 1,
     apis: RUNTIME_API_VERSIONS,
     transaction_version: 26,
```
Comment (Chralt98) [inline runtime/battery-station/src/lib.rs:108]: Double-check: Sure we don't have anything that changes the `transaction_version`?

**Review by Chralt98 — COMMENTED** [review]
I don't get your description for this PR: 

> Merges main into a clone of [branch release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/release-v0.4.3) as a preparation to merge [branch release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/release-v0.4.3) into main.

From the PR I derive:

Merges a clone [sea212-merge-release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/sea212-merge-release-v0.4.3) into main as a preparation to merge [branch release-v0.4.3](https://github.com/zeitgeistpm/zeitgeist/tree/release-v0.4.3) into main.

**Comment by sea212** [comment]
@Chralt98 This PR creates a clone of `release-v0.4.3` such that it can merge the changes contained in `release-v0.4.3` into main without altering the commit history of `release-v0.4.3`, as it is the official release branch.

**Review by Chralt98 — APPROVED** [review]

**Review by maltekliemann — APPROVED** [review]

**Comment by sea212** [comment]
@Mergifyio update

**Pull request merged**